metadata/society/ems_response_areas.md

```diff
@@ -8,15 +8,19 @@ f66983de-8c18-487e-949b-18bb21c3ad6d
 
 ## Brief Summary
 
-Polygon dataset of areas of responsibility for emergency medical services providers.
+Polygon dataset of areas of responsibility for emergency medical services (EMS) providers.
 
 ## Summary
 
+This dataset represents areas of responsibility for emergency medical services (EMS) providers as polygons. This layer is maintained by UGRC in partnership with local government authorities.
+
+<!--- Do we maintain this dataset with other authorities or groups, or should I just keep it to 'local government authorities'? Are we the ones maintaining this dataset at all? --->
+
```

Comment:
We are the sole owner of this dataset and others let us know if changes need to be made.